2018-03-28/azblob/zt_examples_test.go

```diff
@@ -1082,7 +1087,7 @@ func ExampleLeaseContainer() {
 	accountName, accountKey := accountInfo()
 
 	// Use your Storage account's name and key to create a credential object; this is used to access your account.
-	credential := azblob.NewSharedKeyCredential(accountName, accountKey)
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
```

Comment:
given this is an example (and thus a recommended way of using this library) - shouldn't this be checking that the error object returned is nil?